File: Ass2/t2.cpp
#include <stdio.h>
int exp(int number, int exponent);//Calculate the result of 2^n or 3^n
int main()
{
	int n;//The exponent number
	puts("Please input n: ");
	scanf("%d", &n);
	if (n < 0)
	{
		puts("Please input a natural number.");
		return -1;
	}
	printf("The result of 2^%d + 3^%d is %d.\n", n, n, exp(2, n) + exp(3, n));
	return 0;
}
int exp(int number, int exponent)
{
	int result = 1;
	for (int i = 0; i < exponent; i++)
		result *= number;
	return result;
}

File: Ass2/t2.java
import java.util.Scanner;
import java.text.DecimalFormat;
public class t2{
    public static void main(String[] args) {
        double n;
        DecimalFormat form = new DecimalFormat();
        Scanner scan = new Scanner(System.in);
        System.out.println("Please input n: ");
        n = scan.nextDouble();
        if (n < 0){
            System.out.println("Please input natural number.");
            System.exit(-1);
        }
        scan.close();
        System.out.printf("The result of 2^%s + 3^%s is %s.\n", form.format(n), form.format(n), form.format(Math.pow(2, n) + Math.pow(3, n)));
    }
}